erpnext/buying/doctype/supplier/supplier.json

```diff
@@ -16,6 +15,7 @@
   "country",
   "default_bank_account",
   "tax_id",
+  "irs_1099",
```

Comment:
Don't add this in the main JSON, this is conditional to the company being based out of the US